packages/app/src/pages/popup/receive.tsx

```diff
@@ -1,11 +1,51 @@
-import React from 'react';
-import { Box, Text, Button, useClipboard } from '@stacks/ui';
+import React, { memo } from 'react';
+import { Box, Button, color, useClipboard, Flex, FlexProps, Stack } from '@stacks/ui';
 import { PopupContainer } from '@components/popup/container';
 import { useAnalytics } from '@common/hooks/use-analytics';
 import { ScreenPaths } from '@store/onboarding/types';
 import { useWallet } from '@common/hooks/use-wallet';
 import { Toast } from '@components/toast';
-import QRCode from 'qrcode.react';
+import { getAccountDisplayName } from '@stacks/wallet-sdk';
+import { Caption, Title } from '@components/typography';
+
+// eslint-disable-next-line no-warning-comments
+// TODO: fix types
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import vkQr from '@vkontakte/vk-qr';
```

Comment:
Would you like me to add a simple declaration for this?

Reply:
Yes, please!

Reply:
lmao the package is typescript but they have a typo in the package.json, classic

Reply:
@hstove is this resolved?

Reply:
Yep!